packages/airnode-validator/src/config/config.ts

```diff
@@ -106,8 +107,15 @@ export const nodeSettingsSchema = z.object({
   cloudProvider: localOrCloudProviderSchema,
   logFormat: logFormatSchema,
   logLevel: logLevelSchema,
-  // TODO: This must match validator version
-  nodeVersion: z.string(),
+  nodeVersion: z.string().superRefine((version, ctx) => {
```

Comment:
FYI: We should be using `superRefines` in favor of `refine` because the error message from `refine` is too generic.